app/jobs/parse_wdl_workflow.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
```

Comment:
Can you add an header with a description of the script does? Also maybe some comments on inputs and outputs?
Would be great to identify any assumptions on the inputs and why we need them. For instance, is `StageInput` a convention of our pipeline?